### DISCUSSION TITLE
How to make the link active not just for "/docs/components/accordion/" I also want it to be active on  "/docs/components/accordion/api"

### DISCUSSION THREAD

**Discussion by ervinpiol** [fuma-nama/fumadocs#620] [Q&A]
![image](https://github.com/user-attachments/assets/59727764-6f2e-47f7-8a9f-19cb72e43629)
![image](https://github.com/user-attachments/assets/fe455b16-ac86-4ff1-8a66-dcc893664d6e)
![image](https://github.com/user-attachments/assets/8a29a40a-9a9f-47fb-99c6-729a63e65405)
![image](https://github.com/user-attachments/assets/e267a1cc-761a-4109-9e43-fee033931345)

**Comment by fuma-nama** [comment]
Not sure how you managed to make such a page-in-page structure but you should use a folder here, or simply don't affect the pathname when switching between tabs
